llvm/test/CodeGen/EraVM/selectcc.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 2
```

Comment:
I chose to generate checks using a command instead of writing them manually because it's more convenient. If you prefer, I can revert this change, but it could make future modifications easier.